migrations/sql/R__stored_procedure_for_permit_and_permitee_information.sql

```diff
@@ -583,7 +583,7 @@ CREATE OR REPLACE FUNCTION transfer_permit_permitee_information() RETURNS void A
             insert_amendment_row integer;
             total_amendment_row  integer;
         BEGIN
-            RAISE NOTICE '.. Step 2 of 4: Update permit info';
+            RAISE NOTICE '.. Step 2 of 4: Update permit info in MDS';
```

Comment:
We have a large change to permit and permittee - I think you should exclude this whole file from your PR until that work goes in.

Reply:
@itchison Done :D

Reply:
I reverted my changes but the file is still showing due to some non-harmful spacing